techniques/system/common/1.0/promises.st

```diff
@@ -790,17 +790,16 @@ bundle edit_line enforce_content(str)
 bundle edit_line fix_syslogd(syslogd)
 {
   delete_lines:
-    any::
-      "^(local6)\s+(?!${syslogd}).*"
-        comment => "Delete missconfigured rudder syslogd destination";
-      "^local6\.\*\s+${syslogd}"
-        comment => "Delete old rudder syslogd format";
+
+      "^\s*local6.*"
```

Comment:
Possibly. But it means that in this case they would also be jamming the Rudder server with useless (for us) logs if their utility outputs logs in local6.notice.

There are plenty of other facilities to use, I prefer to block logs and make things clear (they will notice the local6.* gets removed) that letting things happen an provoke hard to debug situations.